app/src/views/OperationalLearning/OperationalLearningMap/index.tsx

```diff
@@ -0,0 +1,245 @@
+    learning: learningStatsResponse | undefined;
+ }
+
+function OperationalLearningMap(props: Props) {
+    const strings = useTranslation(i18n);
+    const {
+        className,
+        learning,
+    } = props;
+
+    const [
+        clickedPointProperties,
+        setClickedPointProperties,
+    ] = useState<ClickedPoint | undefined>();
+
+    const countryResponse = useCountry();
+    const countryCentroidGeoJson = useMemo(
+        (): GeoJSON.FeatureCollection<GeoJSON.Geometry> => {
+            const features = countryResponse
+                ?.map((country) => {
+                    const learningList = learning?.learning_by_country?.find(
+                        (item: { country_id: number; }) => item.country_id === country.id,
+                    );
+                    if (isNotDefined(learningList)) {
+                        return undefined;
+                    }
+                    const units = learningList.count;
+                    return {
+                        type: 'Feature' as const,
+                        geometry: country.centroid as {
+                        type: 'Point',
+                        coordinates: [0, 0],
+                    },
+                        properties: {
+                            country_id: country.id,
+                            name: country.name,
+                            operation_count: units,
+                        },
+                    };
+                })
+                .filter(isDefined) ?? [];
+            return {
+                type: 'FeatureCollection',
+                features,
+            };
+        },
+        [countryResponse, learning],
+    );
+
+    const bluePointHaloCirclePaint = useMemo(() => {
+        const learningCount = learning?.learning_by_country
+            .filter((country) => country.count > 0);
+
+        const maxScaleValue = Math.max(
+            ...(learningCount
+                ?.map((activity: { count: number; }) => activity.count)
+                .filter(isDefined) ?? []),
+        );
+
+        return getPointCircleHaloPaint(COLOR_BLUE, 'operation_count', maxScaleValue);
+    }, [learning]);
+
+    const handleCountryClick = useCallback(
+        (feature: mapboxgl.MapboxGeoJSONFeature, lngLat: mapboxgl.LngLatLike) => {
+            setClickedPointProperties({
+                feature: feature as unknown as ClickedPoint['feature'],
+                lngLat,
+            });
+            return false;
+        },
+        [],
+    );
+    const handlePointClose = useCallback(() => {
+        setClickedPointProperties(undefined);
+    }, []);
+
+    const handlePointClick = useCallback(
+        (feature: mapboxgl.MapboxGeoJSONFeature, lngLat: mapboxgl.LngLatLike) => {
+            setClickedPointProperties({
+                feature: feature as unknown as ClickedPoint['feature'],
+                lngLat,
+            });
+            return true;
+        },
+        [setClickedPointProperties],
+    );
```

Comment:
Not sure if handleCountryClick and handlePointClick can be handled the same way. Do we even need handleCountryClick?